### PR TITLE
Update to align with startos package template

### PR DESCRIPTION
**Pull request by islandbitcoin**
The package wrapper works, now the webserver spins up and runs on port 8080.

```
2023-10-20T14:01:50-04:00  2023/10/20 18:01:50 [notice] 134#134: start worker process 163
2023-10-20T14:01:50-04:00  2023/10/20 18:01:50 [notice] 134#134: start worker process 164
2023-10-20T14:01:50-04:00  Webserver running on http://0.0.0.0:8080
```

But I see this in the UI `No route for /`

so your nginx file might need some modifying or maybe the docker_entrypoint.sh, not sure (this is Mutiny code now)

Hope this helps

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -44,10 +50,10 @@ ARG VITE_SELFHOSTED="true"
 RUN pnpm install --frozen-lockfile
 
 # IDK why but it gets mad if you don't do this
-RUN git config --global --add safe.directory /app
+# RUN git config --global --add safe.directory /app
 
 # Build the static site
-RUN pnpm run build
+# RUN pnpm run build
```
Comment (benthecarman) [inline Dockerfile]: it is no longer building the website so that's why you're getting the route not found

Comment (islandbitcoin) [reply Dockerfile]: I tried to put that in the docker_entrypoint

Comment (benthecarman) [reply Dockerfile]: made some updates that got it working for me with the submodules

Comment (benthecarman) [reply Dockerfile]: you'll need to run `git submodule update`, we had to make a small change to mutiny-web

Comment (islandbitcoin) [reply Dockerfile]: works ™

**Pull request merged**